sim/core/character.go

```diff
@@ -374,16 +371,21 @@ func (character *Character) GetBaseStats() stats.Stats {
 // https://github.com/TheGroxEmpire/TBC_DPS_Warrior_Sim/issues/30
 // TODO "primaryModifiers" could be modelled as a PseudoStat, since they're unit-specific. "secondaryModifiers" apply to a specific set of spells.
 func (character *Character) calculateCritMultiplier(normalCritDamage float64, primaryModifiers float64, secondaryModifiers float64) float64 {
-	if character.HasMetaGemEquipped(34220) ||
-		character.HasMetaGemEquipped(32409) ||
-		character.HasMetaGemEquipped(41285) ||
-		character.HasMetaGemEquipped(41376) ||
-		character.HasMetaGemEquipped(41398) ||
-		character.HasMetaGemEquipped(52291) ||
+	// Cata
+	if character.HasMetaGemEquipped(52291) ||
 		character.HasMetaGemEquipped(52297) ||
 		character.HasMetaGemEquipped(68778) ||
 		character.HasMetaGemEquipped(68779) ||
-		character.HasMetaGemEquipped(68780) {
+		character.HasMetaGemEquipped(68780) ||
+		// MOP
+		character.HasMetaGemEquipped(76884) ||
+		character.HasMetaGemEquipped(76885) ||
+		character.HasMetaGemEquipped(76886) ||
+		character.HasMetaGemEquipped(76884) ||
+		character.HasMetaGemEquipped(76888) ||
+		character.HasMetaGemEquipped(97937) ||
+		character.HasMetaGemEquipped(97534) ||
+		character.HasMetaGemEquipped(97306) {
 		primaryModifiers *= 1.03
 	}
```

Comment:
Now that we have the `PseudoStat` for multiplicative effects, this code should be moved from here to `applyItemEffects()`, where you can apply the same 1.03 multiplier directly to `character.PseudoStats.CritDamageMultiplier` if one of these meta gems is equipped.